requirements/test.txt

```diff
@@ -1,7 +1,7 @@
 allure-pytest
 coverage
 pynvml
-pytest==8.0.2
+pytest
```

Comment:
why unlock the version of pytest? Is the version issue resolved?

Reply:
> why unlock the version of pytest? Is the version issue resolved?

 this condition is added for bug on version 8.1 https://github.com/pytest-dev/pytest/issues/12069
this bug is fixed, I want to remove the restriction,